js/_current-weather.js

```diff
@@ -127,22 +127,59 @@ async function displayCurrentWeatherData(data) {
   //     `
   // weatherDataElement.innerHTML = weatherInfo
   // Update City Header
+  console.log(data)
```

Comment:
Before pushing your commit, we need to make sure and delete unnecessary debug code unless it's intentionally placed.

I'd appreciate it if you could also remove the comment out section above as well.